packages/console/src/components/RolesTransfer/components/RoleInformation/index.tsx

```diff
@@ -9,19 +15,36 @@ type Props = {
 
 function RoleInformation({ role }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { id, name, type, usersCount, applicationsCount } = role;
+  const { data } = useSWR<ScopeResponse[]>(
+    // Todo @xiaoyijun remove dev feature flag
+    isDevFeaturesEnabled && type === RoleType.MachineToMachine && `api/roles/${id}/scopes`
+  );
 
-  const { name, type, usersCount, applicationsCount } = role;
+  const withManagementApiAccess = data?.some(({ resource: { indicator } }) =>
+    isManagementApi(indicator)
+  );
 
   return (
     <div className={styles.container}>
       <div className={styles.name}>{name}</div>
-      <div className={styles.count}>
-        (
-        {type === RoleType.User
-          ? t('user_details.roles.assigned_user_count', { value: usersCount })
-          : t('application_details.roles.assigned_app_count', { value: applicationsCount })}
-        )
-      </div>
+      {!isDevFeaturesEnabled && (
+        <div className={styles.count}>
+          (
+          {type === RoleType.User
+            ? t('user_details.roles.assigned_user_count', { value: usersCount })
+            : t('application_details.roles.assigned_app_count', { value: applicationsCount })}
+          )
+        </div>
+      )}
+      {withManagementApiAccess && (
+        <Tooltip
+          anchorClassName={styles.flag}
```

Comment:
Could you try to name it, like `managementApiFlag`.

Reply:
Updated.